encoding/yaml/Mark.ts

```diff
@@ -0,0 +1,72 @@
+import { repeat } from "./utils.ts";
```

Comment:
There are some files with capitalized/camelCase style, can you change it to snake_case to keep it identical with rest of repo?

Reply:
All files are now snake_cased.